
float32 roll_rate_integ	    # roll rate inegrator
float32 pitch_rate_integ	# pitch rate integrator
float32 yaw_rate_integ	    # yaw rate integrator
float32 roll_rate_foaw_deriv
float32 pitch_rate_foaw_deriv
float32 yaw_rate_foaw_deriv
float32 dt
uint8 window_size
